packages/iw/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://wireless.wiki.kernel.org/en/users/documentation/iw
+TERMUX_PKG_DESCRIPTION="CLI configuration utility for wireless devices"
+TERMUX_PKG_VERSION=4.9
+TERMUX_PKG_SRCURL=https://www.kernel.org/pub/software/network/iw/iw-${TERMUX_PKG_VERSION}.tar.xz
+TERMUX_PKG_SHA256=324cc805fad52cba2c16b9ab569906889fb645cc962aac4cfda1db85d2de97ce
+TERMUX_PKG_MAINTAINER="Auxilus @Auxilus"
+TERMUX_PKG_DEPENDS="libnl, libnl-dev, pkg-config"
+TERMUX_PKG_BUILD_IN_SRC=yes
+
+termux_step_post_configure () {
+      sed 's|SBINDIR|BINDIR|g' -i Makefile
+      sed 's|sbin|bin|g' -i Makefile
```

Comment:
See the comment in https://github.com/termux/termux-root-packages/pull/9 - the build script already moves over files from sbin/ to bin/, so perhaps this is not needed?